.github/workflows/weekly_vuln_scan.yml

```diff
@@ -1,41 +1,60 @@
+# SPDX-FileCopyrightText: 2024 PNED G.I.E.
+# SPDX-License-Identifier: Apache-2.0
+
 name: Scan Latest Two Semantically Versioned Docker Image Tags with ORT
 
 on:
   schedule:
-    - cron: "0 14 * * 1" # Every Monday at 14:00 UTC
+    - cron: "0 14 * * 1" # Works on Every Monday at 14:00 UTC
   workflow_dispatch:
 
 jobs:
   fetch-and-scan:
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout repository
```

Comment:
question: why do you need to do checkout of the repo, if we are scanning only docker images?

Reply:
It's not necessary actually, I've added it just because if any case needs repo based analysis. However it's not necessary, let me remove this part.